src/test/fuzz/kitchen_sink.cpp

```diff
@@ -46,11 +47,9 @@ FUZZ_TARGET(kitchen_sink)
 
     const OutputType output_type = fuzzed_data_provider.PickValueInArray(OUTPUT_TYPES);
     const std::string& output_type_string = FormatOutputType(output_type);
-    OutputType output_type_parsed;
-    const bool parsed = ParseOutputType(output_type_string, output_type_parsed);
-    assert(parsed);
-    assert(output_type == output_type_parsed);
-    (void)ParseOutputType(fuzzed_data_provider.ConsumeRandomLengthString(64), output_type_parsed);
+    const std::optional<OutputType> parsed = ParseOutputType(output_type_string);
+    assert(output_type == parsed.value());
```

Comment:
We should assert that `parsed` has a value before extracting the object:

```suggestion
    assert(parsed);
    assert(output_type == parsed.value());
```

Reply:
This just round-trips known values, so I don't think it can ever not have a value. However we can re-add the assert.